RecoLocalCalo/HGCalRecAlgos/src/ClusterTools.cc

```diff
@@ -83,14 +85,16 @@ math::XYZPoint ClusterTools::getMultiClusterPosition(const reco::HGCalMultiClust
   for( const auto& ptr : clu.clusters() ) {   
     const double x = ptr->x();    
     const double y = ptr->y();
-    const float point_r = std::sqrt(x*x + y*y);    
+    const double point_r2 = (x*x + y*y);    
```

Comment:
everything else is already in double, repeated conversion from double to float costs time.